lib/packages/core/src/typings/api/networkApi.ts

```diff
@@ -12,19 +14,33 @@ import {SuggestedFees} from '../suggestedFees';
 export interface NetworkApi {
     /**
      * Get the blockchain status.
-     * @return The Blockchain Status
+     * @return {BlockchainStatus} The Blockchain Status
```

Comment:
with the tsdoc tool you shouldn't need the typings

Reply:
oh, very nice 👯‍♂️